lib/src/main/java/com/futurewei/alcor/common/db/ignite/IgniteTransaction.java

```diff
@@ -33,55 +31,43 @@
 public class IgniteTransaction implements Transaction {
     private static final Logger logger = LoggerFactory.getLogger();
 
-    private IgniteClient igniteClient;
-    private ClientTransaction clientTransaction;
+    private final Ignite client;
+    private org.apache.ignite.transactions.Transaction transaction;
 
-    public IgniteTransaction(IgniteClient igniteClient) {
-        this.igniteClient = igniteClient;
+    public IgniteTransaction(Ignite client) {
+        this.client = client;
     }
 
+    @Override
     public Transaction start() throws CacheException {
```

Comment:
Do we still need throws CacheException here?

Reply:
IgniteClienTransaction need